### PR TITLE
feat: drop flows

### PR DESCRIPTION
**Pull request by colbr**
[ZAP-2](https://wilderworld.atlassian.net/browse/ZAP-2?atlOrigin=eyJpIjoiMWFlMzAxZWY0YjkxNDRjM2E4ZWUzYzMwNThmZjA3MmUiLCJwIjoiaiJ9)

[ZAP-2]: https://wilderworld.atlassian.net/browse/ZAP-2?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/features/mint-nft-drop/components/DropBanners/DropBanners.module.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+.BannerContainer {
+    position: relative;
+    margin: 0 auto;
+    // margin-top: 25px;
+    text-align: center;
+
+    padding: 5.375rem 2rem 1rem 2rem;
+
+    // Desktop Breakpoint
+    @media only screen and (min-width: $breakpoint-desktop-s) {
```
Comment (domw30) [inline src/features/mint-nft-drop/components/DropBanners/DropBanners.module.scss:12]: could remove this media query and comments

File: src/features/mint-nft-drop/components/DropForm/DropForm.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	}, [balanceEth]);
+
+	const getFlowSection = () => {
+		if (props.dropStage === undefined) {
+			return;
+		}
+		if (step === Step.Info) {
+			return (
+				<Info
+					onContinue={() => console.log('yo')}
```
Comment (domw30) [inline src/features/mint-nft-drop/components/DropForm/DropForm.tsx:94]: Should these be console.logs?

File: src/features/mint-nft-drop/lib/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+	zSaleInstance: GenSaleInstance,
+): Promise<boolean | undefined> => {
+	if (TEST_MODE) {
+		return IS_ON_WHITELIST;
+	} else {
+		return await zSaleInstance.isUserOnMintlist(account);
+	}
+};
+
+const getWheelQuantities = async (
```
Comment (domw30) [inline src/features/mint-nft-drop/lib/helpers.ts:147]: I'd probably rename this to something generic :)

**Review by domw30 — DISMISSED** [review]
Hey @colbr. I had a quick flick through of this. I agree, lets merge it in and then come back to it when we can test with a up coming drop. Looks like some more refactoring can be done but I don't think the time spent on it right now would be valuable. Nice one, that's a tough one to migrate over! 🚀

File: src/features/mint-nft-drop/components/DropForm/DropForm.tsx (hunk 2 above)
Comment (colbr) [reply src/features/mint-nft-drop/components/DropForm/DropForm.tsx:94]: Oops - forgot to pick these up. Changed to `console.warn`

**Review by domw30 — APPROVED** [review]
Looks good mate! 💯

**Pull request merged**